### PR TITLE
[ZT] MASQUE

### PR DESCRIPTION
**Pull request by ranbel**
PCX-9869

### CODE REVIEW

File: content/cloudflare-one/connections/connect-devices/warp/configure-warp/warp-settings/_index.md [hunk 1]
```diff
@@ -106,6 +106,34 @@ Since captive portal implementations vary, WARP may not detect all captive porta
...
+
+**Value**:
+
+- **WireGuard**: (default) Establishes a [WireGuard](https://www.wireguard.com/) connection to Cloudflare. The WARP client will encrypt traffic using a non-FIPs compliant cipher suite, `TLS_CHACHA20_POLY1305_SHA256`. When switching from MASQUE to WireGuard, users may lose Internet connectivity if their Wi-Fi network blocks the [ports and IPs](/cloudflare-one/connections/connect-devices/warp/deployment/firewall/#warp-ingress-ip) required for WireGuard to function.
+- **MASQUE** {{<inline-pill style="beta">}}: Establishes an HTTP/3 connection to Cloudflare. To use MASQUE, [Override local interface IP](/cloudflare-one/connections/connect-devices/warp/configure-warp/warp-settings/#override-local-interface-ip) must be `Enabled`. The WARP client will encrypt traffic using TLS 1.3 and a [FIPS 140-2](https://csrc.nist.gov/pubs/fips/140-2/upd2/final) compliant cipher suite, `TLS_AES_256_GCM_SHA384`. Since AES-GCM is a more computationally intensive algorithm, users may experience lower data rates on MASQUE compared to WireGuard. Many modern CPUs support AES-GCM acceleration, which will mitigate the performance impact of switching to MASQUE.
```
Comment (kkrum) [inline content/cloudflare-one/connections/connect-devices/warp/configure-warp/warp-settings/_index.md]: I don't think we should include the perf impact lines at the end. Do we have any data to justify this? In all the test results I've seen or collected we see more throughput and lower latency.

File: content/cloudflare-one/connections/connect-devices/warp/configure-warp/warp-settings/captive-portals.md [hunk 2]
```diff
@@ -12,7 +12,13 @@ Captive portals are used by public Wi-Fi networks (such as airports, coffee shop
 
 To allow users to connect through a captive portal, administrators can configure the following WARP settings:
 
+### No user interaction required
+
 - Enable [Captive portal detection](/cloudflare-one/connections/connect-devices/warp/configure-warp/warp-settings/#captive-portal-detection). This allows WARP to temporarily turn off when it detects a captive portal on the network. For more details, refer to [how captive portal detection works](#how-captive-portal-detection-works) and its [limitations](#limitations).
+- Set [Device tunnel protocol](/cloudflare-one/connections/connect-devices/warp/configure-warp/warp-settings/#device-tunnel-protocol) to **MASQUE**. When using MASQUE, WARP traffic will look like standard HTTPS traffic and is therefore very unlikely to be blocked by captive portals.
```
Comment (kkrum) [inline content/cloudflare-one/connections/connect-devices/warp/configure-warp/warp-settings/captive-portals.md]: Was there a discussion on "very unlikely" vs. stating "less likely"? I am concerned about making a promise w e have little data on or control of.

**Review by kkrum — COMMENTED** [review]
Few comments we should address or discuss before posting live

File: content/cloudflare-one/connections/connect-devices/warp/configure-warp/warp-settings/captive-portals.md (hunk 2 above)
Comment (ranbel) [reply content/cloudflare-one/connections/connect-devices/warp/configure-warp/warp-settings/captive-portals.md]: "Very unlikely" was copied from our blog post: https://blog.cloudflare.com/zero-trust-warp-with-a-masque

@dh-cf do you think we should tone this down?

File: content/cloudflare-one/connections/connect-devices/warp/configure-warp/warp-settings/_index.md (hunk 1 above)
Comment (ranbel) [reply content/cloudflare-one/connections/connect-devices/warp/configure-warp/warp-settings/_index.md]: We added this to be on the safe side, same reason as the "24 hours" statement above. Can we confidently say that there is no perf impact?

File: content/cloudflare-one/connections/connect-devices/warp/configure-warp/route-traffic/warp-architecture.md [hunk 3]
```diff
@@ -175,7 +175,7 @@ S -- No --> U["Virtual interface<br> (172.16.0.2)"] --> G[Cloudflare Gateway]
 
 #### Virtual interface
 
-Virtual interfaces allow the operating system to logically subdivide a physical interface, such as a network interface controller (NIC), into separate interfaces for the purposes of routing IP traffic. WARP’s virtual interface is what maintains the Wireguard connection between the device and Cloudflare. By default, its IP address is hardcoded as `172.16.0.2`. You can optionally use [**Override local interface IP**](/cloudflare-one/connections/connect-devices/warp/configure-warp/warp-settings/#override-local-interface-ip) to assign unique IPs per device.
+Virtual interfaces allow the operating system to logically subdivide a physical interface, such as a network interface controller (NIC), into separate interfaces for the purposes of routing IP traffic. WARP’s virtual interface is what maintains the WireGuard/MASQUE connection between the device and Cloudflare. By default, its IP address is hardcoded as `172.16.0.2`. You can optionally use [**Override local interface IP**](/cloudflare-one/connections/connect-devices/warp/configure-warp/warp-settings/#override-local-interface-ip) to assign unique IPs per device.
```
Comment (aw-cf) [inline content/cloudflare-one/connections/connect-devices/warp/configure-warp/route-traffic/warp-architecture.md]: "Override local interface IP" will be required for MASQUE

**Review by crwaters16 — APPROVED** [review]

File: content/cloudflare-one/connections/connect-devices/warp/configure-warp/warp-settings/_index.md [hunk 4]
```diff
@@ -106,6 +106,34 @@ Since captive portal implementations vary, WARP may not detect all captive porta
...
+
+| [WARP modes](/cloudflare-one/connections/connect-devices/warp/configure-warp/warp-modes/) | [Zero Trust plans](https://www.cloudflare.com/teams-pricing/) |
+| -- | -- |
+| <ul><li> Gateway with WARP</li><li> Secure Web Gateway without DNS filtering </li></ul>| All plans  |
+
+| System   | Availability | Minimum WARP version |
+| ---------| -------------| ---------------------|
+| Windows  | ✅           | 2024.6.415.0      |
+| macOS    | ✅           | 2024.6.416.0      |
+| Linux    | Coming soon           |      |
```
Comment (ranbel) [inline content/cloudflare-one/connections/connect-devices/warp/configure-warp/warp-settings/_index.md]:
```suggestion
| Linux    | Coming soon           |       |
```

**Pull request merged**